includes/class-activate.php

```
@@ -108,18 +104,28 @@ private static function add_wp_cache_constant(): ?string {
+				$insert_position = strpos( $wp_config_content, "/* That's all, stop editing!" );
 
-			if ( false !== $insert_position ) {
-				$wp_config_content = substr_replace( $wp_config_content, $constant_code, $insert_position, 0 );
-			} else {
-				$wp_config_content .= $constant_code;
+				if ( false !== $insert_position ) {
+					$wp_config_content = substr_replace( $wp_config_content, $constant_code, $insert_position, 0 );
+				} else {
+					$wp_config_content .= $constant_code;
+				}
 			}
```

PHPMD (2.15.0)

[error] The method add_wp_cache_constant uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)